config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -189,20 +242,20 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 24h
+- interval: 6h
```

Comment:
unsure why the lower interval

Reply:
blame amit :-) done

Reply:
/shrug this is a combination of both a bad git diff plus not knowing why we run the various stableX jobs on different intervals.